### ISSUE TITLE
SPP REST API errors beginning with SPP 10.1.8.1

### ISSUE THREAD

**Issue by baslerj** [IBM/spectrum-protect-sppmon#64]
After upgrading SPP to 10.1.8.1, the following API error is reported:
```
> Testing REST-API of SPP.
Exception in FILE: other.py, Line: 53, Exception: <class 'ValueError'>
Exception Message: Wrong Status code when requesting endpoint data
Exception args: (500, 'https://x.x.x.x:443/ngp/version?pageSize=50', <Response [500]>)
> Testing of the REST-API failed. This is a crictial component of SPPMon.
```

**Comment by baslerj** [comment]
I have confirmed the ngp/version API endpoint is failing independent of SPPmon.  There is an alternate /api/lifecycle/ping endpoint which provides the same version.  I will produce a hotfix with exception handling to use the alternate endpoint while we sort out which API should be used.

**Comment by NielsKorschinsky** [comment]
solved with #65 and then merged with dev into master #66
